build.gradle

```diff
@@ -73,6 +73,7 @@ subprojects {
     }
     compile 'org.apache.commons:commons-text:1.6'
     compile 'org.apache.commons:commons-lang3:3.8.1'
+    compile 'commons-collections:commons-collections:3.2.2'
```

Comment:
This dependency is unused and should be removed. It is also an older version of commons-collections.